docs/AutomationProtocols.md

```diff
@@ -0,0 +1,98 @@
+---
+id: automationProtocols
+title: Automation Protocols
+---
+
+With WebdriverIO you can choose between multiple automation technologies when running your e2e tests locally or in the cloud. By now almost all modern browser support next to [WebDriver](https://w3c.github.io/webdriver/) also another native interface called [DevTools](https://chromedevtools.github.io/devtools-protocol/) that can be used for automation purposes. Both have advantages and disadvantages depending on your use case and environment.
+
+## WebDriver Protocol
+
+> [WebDriver](https://w3c.github.io/webdriver/) is a remote control interface that enables introspection and control of user agents. It provides a platform- and language-neutral wire protocol as a way for out-of-process programs to remotely instruct the behavior of web browsers.
+
+The WebDriver protocol was designed to automated a browser from the user perspective meaning that everything a user is able to do you can do with the browser. It provides a set of commands that abstracts away common interaction with an application (e.g. navigating, clicking or reading the state of an element). Since it is a web standard it is well supported across all major browser vendors and also is being used as underlying protocol for mobile automation using [Appium](http://appium.io/).
+
+In order to use this automation technology you will need a proxy server that translates all commands and executes them in the target environment (e.g. the browser or the mobile app). For browser automation this proxy server is usually a browser driver that is available for all browser:
```

Comment:
`that is available for all browsers` instead of `that is available for all browser`